### PR TITLE
[o365_audit] Add NewValue field

### PR DESCRIPTION
**Pull request by Mikaayenson**
99435618-5c38a900-28de-11eb-9fdd-32cdc7d31196

Adds the `o365.audit.NewValue` field. It is present in the filebeat [pipeline.js](https://github.com/elastic/beats/blob/main/x-pack/filebeat/module/o365/audit/config/pipeline.js#L136-L153) and available for use in a query (per the screenshot), but not defined in either [Beats](https://github.com/elastic/beats/blob/main/x-pack/filebeat/module/o365/audit/_meta/fields.yml) or the Elastic Agent [integration](https://github.com/elastic/integrations/blob/main/packages/o365/data_stream/audit/fields/fields.yml) mappings.

## What does this PR do?

Updates the `fields.yml` to include the field.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

**Review by kcreddy — APPROVED** [review]
LGTM

File: packages/o365/manifest.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: o365
 title: Microsoft 365
-version: "1.10.1"
+version: "1.10.2"
```
Comment (Mikaayenson) [inline packages/o365/manifest.yml:3]: @kcreddy Does this have to be a minor bump since it's an enhancement?

Comment (efd6) [reply packages/o365/manifest.yml:3]: Yes please.

File: packages/o365/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.10.2"
```
Comment (Mikaayenson) [inline packages/o365/changelog.yml:2]:
```suggestion
- version: "1.11.0"
```

**Comment by Mikaayenson** [comment]
@efd6 Anything else on this PR?

**Review by efd6 — APPROVED** [review]
Thanks

**Comment by Mikaayenson** [comment]
👋 @efd6 For this the changelog shows that the minor has been already bumped by another Pr. Should I update the minor again or patch version?

**Comment by efd6** [comment]
Bump the minor, 1.12.0.

**Pull request merged**